distros/arch/config

```diff
@@ -1,4 +1,4 @@
-URL=https://github.com/dreemurrs-embedded/Pine64-Arch/releases/download/20230925/archlinux-pinephone-pro-phosh-20230925.img.xz
+URL=https://github.com/dreemurrs-embedded/Pine64-Arch/releases/download/20240308/archlinux-pinephone-pro-phosh-20240308.img.xz
 PARTLABEL=ppp-multi-image-arch
```

Comment:
Since "The shell script in the repository above was derived from [these instructions](https://pine64.org/documentation/PinePhone_Pro/Software/Multi-distribution_image/).", do you think is possible to align PARTLABEL name to better inter-operate between the projects?

PARTLABEL=ppp-multi-image-arch
vs
PARTLABEL=ARCH

In case to adapt for all other ditros.

thanks

Reply:
No, I did that intentionally because the script accesses the partitions via `/dev/disk/by-partlabel/` and that mechanism cannot differentiate between partitions with the same partition label.

So if the computer you're running the shell script on has its root filesystem on a partition with `PARTLABEL=ARCH` guess what can happen..

Reply:
so, in the https://pine64.org/documentation/PinePhone_Pro/Software/Multi-distribution_image/#partitioning block it should be better to rename PARTLABEL=ARCH into PARTLABEL=ppp-multi-image-arch?
thanks for explaning

Reply:
Up to you. Bear in mind that your instructions won't have the same issue since they don't use the `/dev/disk/by-partlabel/` mechanism.